apps/nextjs/src/stores/AilaStoresProvider.tsx

```diff
@@ -3,7 +3,7 @@ import { createContext, useContext, useEffect, useRef, useState } from "react";
 import invariant from "tiny-invariant";
```

Comment:
This file is where the main changes start to happen - putting the store in the context